docs/source/upgrade_notes.rst

```diff
@@ -3,6 +3,21 @@
 Upgrade Notes
 =============
 
+.. _ref-upgrade-notes-v3-6:
+
+|st2| v3.6
+----------
+
+* With kept backward compatibility, action delete API has been modified.
```

Comment:
Perhaps rephrase " With kept backward compatibility, action delete API has been modified." to " Retaining backwards compatibility, action delete API has been modified."